css/style.css

```diff
@@ -7,58 +7,74 @@
   font-family: "Inter", sans-serif;
 }
 
-a {
-  text-decoration: none;
-  color: inherit;
-}
+/* Main Body Style Canvas . have to EDIT */
 
-.portfolio {
+/* .portfolio {
   background: white;
-  background-image: url('../images/background.png');
+  background-image: url('../images/header-illsutration-mobile@2x.svg');
   background-size: cover;
   background-position: center;
   background-repeat: no-repeat;
   border-radius: 0;
-}
+} */
```

Comment:
- [ ] Kindly remove all commented code 🙏 . This will help make your codebase neater and easier to read.

Reply:
cleaned the comments